### PR TITLE
Add UI Working Group Meeting Notes 26032022

### PR DESCRIPTION
**Pull request by gchhablani**
This PR adds the UI working group weekly meeting notes for 26th March 2022.

@chicken-biryani

### CODE REVIEW

File: working-groups/user-interface/MeetingNotes/26-3-22.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Agenda
+- FLINT-UI Development
+- Tech Stack of FLINT-UI
+- Focus Group
+
+
+## Discussion
+- Focus on Vue
+- Yash will pick up few components:
```
Comment (chicken-biryani) [inline working-groups/user-interface/MeetingNotes/26-3-22.md:18]:
```suggestion
- Yash, Sanjay and other contributors will create a few components:
```

File: working-groups/user-interface/MeetingNotes/26-3-22.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - Slider
+    - Date Picker
+    - Footer
```
Comment (chicken-biryani) [inline working-groups/user-interface/MeetingNotes/26-3-22.md:21]:
```suggestion
```

File: working-groups/user-interface/MeetingNotes/26-3-22.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- Spend the new week looking up Vue: https://vuejs.org/guide/introduction.html
+- Courses & :
+    - https://www.vuemastery.com/courses-path/beginner/
+    - https://www.youtube.com/watch?v=5LYrN_cAJoA&list=PL4cUxeGkcC9gQcYgjhBoeQH7wiAyZNrYa
+- Focus on Vue 2/3
```
Comment (chicken-biryani) [inline working-groups/user-interface/MeetingNotes/26-3-22.md:26]:
```suggestion
- Focus on Vue 2
```

File: working-groups/user-interface/MeetingNotes/26-3-22.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- Figma design - run button, table will be displayed, but table is not displayed.
+    - The task is to make the component.
+
+## Check in on previous action items
+- [ ] Start with RothC and Point from the next meeting.
+- [ ] Sukriti to merge TSC WG PR.
+
+## New Action items
+- [ ] Learn Vue
+- [ ] Discuss findings
```
Comment (chicken-biryani) [inline working-groups/user-interface/MeetingNotes/26-3-22.md:40]:
```suggestion
- [ ] Discuss findings by writing blogs or any other way.
```

**Review by chicken-biryani — CHANGES_REQUESTED** [review]
Hi @gchhablani thank you so much! Can you please do the requested changes?

File: working-groups/user-interface/MeetingNotes/26-3-22.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+# 26-03-2022 UI Working Group weekly meeting 
+
+## Attendees
+- Baibhav
+- Gunjan Chhablani
+- Janvi Thakkar
+- Ritika Lohia
+- Shloka Gupta
+
```
Comment (Sukriti-sood) [inline working-groups/user-interface/MeetingNotes/26-3-22.md:9]: Arpita and me was also in the meet 😅

File: working-groups/user-interface/MeetingNotes/26-3-22.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+# 26-03-2022 UI Working Group weekly meeting 
+
+## Attendees
+- Baibhav
+- Gunjan Chhablani
+- Janvi Thakkar
+- Ritika Lohia
+- Shloka Gupta
```
Comment (chicken-biryani) [inline working-groups/user-interface/MeetingNotes/26-3-22.md:8]:
```suggestion
- Arpita
- Baibhav
- Gunjan Chhablani
- Janvi Thakkar
- Ritika Lohia
- Shloka Gupta
- Sukriti Sood
```

**Pull request merged**